lbrynet/tests/unit/daemon/test_claims_comparator.py

```diff
@@ -0,0 +1,214 @@
+import unittest
+
+from lbrynet.daemon.Daemon import arrange_results
+
+
+class ClaimsComparatorTest(unittest.TestCase):
+    def test_arrange_results_when_sorted_by_claim_id(self):
+        self.run_test(
+            [
+                {
+                    "height": 1,
+                    "name": "res",
+                    "claim_id": "ccc",
+                    "nout": 0,
+                    "txid": "fdsafa"
+                },
+                {
+                    "height": 1,
+                    "name": "res",
+                    "claim_id": "aaa",
+                    "nout": 0,
+                    "txid": "w5tv8uorgt"
+                },
+                {
+                    "height": 1,
+                    "name": "res",
+                    "claim_id": "bbb",
+                    "nout": 0,
+                    "txid": "aecfaewcfa"
+                }
+            ],
+            [
+                {
+                    "height": 1,
+                    "name": "res",
+                    "claim_id": "aaa",
+                    "nout": 0,
+                    "txid": "w5tv8uorgt"
+                },
+                {
+                    "height": 1,
+                    "name": "res",
+                    "claim_id": "bbb",
+                    "nout": 0,
+                    "txid": "aecfaewcfa"
+                },
+                {
+                    "height": 1,
+                    "name": "res",
+                    "claim_id": "ccc",
+                    "nout": 0,
+                    "txid": "fdsafa"
+                }
+            ])
```

Comment:
This is looking much better; you can reduce the code even more if you make the `expected` part above just be a list of `txid`s. So, in above case, the second argument to `run_test` would just be `["w5tv8uorgt", "aecfaewcfa", "fdsafa"]`.

Then in `run_test`:
```python
self.assertEqual(expected, [r['txid'] for r in actual])
```